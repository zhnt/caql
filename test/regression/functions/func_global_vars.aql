let global_counter = 0

function increment() {
    global_counter = global_counter + 1
    print("Global counter: " + string(global_counter))
}

function show_global() {
    print("Current global: " + string(global_counter))
}

show_global()
increment()
increment()
show_global()


