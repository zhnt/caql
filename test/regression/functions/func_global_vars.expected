Current global: 0
Global counter: 1
Global counter: 2
Current global: 2

